tests/foreman/api/test_role.py

```diff
@@ -622,12 +858,78 @@ def test_positive_user_group_users_access_as_org_admin(self):
             3. Create two users without assigning roles while creating them
             4. Assign Organization A and Location A to both users
             5. Create an user group with above two users
+            6. Assign Org Admin role to User Group
 
         :expectedresults: Both the user should have access to the resources of
             organization A and Location A
 
         :CaseLevel: System
         """
+        org_admin = self.create_org_admin(
+            orgs=[self.org1.id], locs=[self.loc1.id])
+        user1_login = gen_string('alpha')
+        user1_pass = gen_string('alphanumeric')
+        user1 = entities.User(
+            login=user1_login,
+            password=user1_pass,
+            auth_source=1,
```

Comment:
This line looks weird, and, basically there's no need in it since auth source with id=1 is automatically populated by nailgun, you may remove this line.